src/components/container/SwellingCalendar/SwellingCalendar.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import Card from "../../presentational/Card"
+import Layout from "../../presentational/Layout"
+import SwellingCalendar from "./SwellingCalendar";
+import { SurveyResultDataParameters } from "../../../helpers/daily-data-providers";
+
+export default {
+    title: "Container/SwellingCalendar",
+    component: SwellingCalendar,
+    parameters: {
+        layout: 'fullscreen',
```

Comment:
Minor: `'` (single quotes) here is inconsistent with the rest of the file.  Also, the extra `,` is not needed at the end.